charts/fluentd/README.md

```diff
@@ -69,10 +69,12 @@ helm install -n monitoring \
 | `apiVersions.configmap` | Configmap API version. | `v1` |
 | `apiVersions.secret` | Secret API version. | `v1` |
 | `namespace` | Chart's namespace. | `monitoring` |
+| `fargateLogRouter.enabled` | Boolen to decide if to configure fargate log router | `false` |
 | `isRBAC` | Specifies whether the Chart should be compatible to a RBAC cluster. If you're running on a non-RBAC cluster, set to `false`.  | `true` |
 | `serviceAccount.name` | Name of the service account. | `""` |
 | `daemonset.tolerations` | Set [tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for all DaemonSet pods. | See [values.yaml](https://github.com/logzio/logzio-helm/blob/master/charts/fluentd/values.yaml). |
 | `daemonset.nodeSelector` | Set [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for all DaemonSet pods. | `{}` |
+| `daemonset.affinity` | Set [affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) for all DaemonSet pods. |  |
```

Comment:
Its not clear what customer should do with this parameter

Reply:
@Doron-Bargo Better now?